subsystem/cluster_test.go

```diff
@@ -1486,6 +1486,25 @@ var _ = Describe("cluster install", func() {
 						defaultWaitForHostStateTimeout)
 				}
 			})
+			It("cancel installation - cluster in finalizing status", func() {
+				c := installCluster(clusterID)
+				Expect(swag.StringValue(c.Status)).Should(Equal("installing"))
```

Comment:
please use consts and not strings (`models.HostStatus...`)
Same comment for the next test